app/controllers/ChoiceController.scala

```diff
@@ -36,10 +41,13 @@ import scala.concurrent.{ExecutionContext, Future}
 class ChoiceController @Inject()(
   authenticate: AuthAction,
   customsCacheService: CustomsCacheService,
+  exportsCacheService: ExportsCacheService,
   errorHandler: ErrorHandler,
   mcc: MessagesControllerComponents
 )(implicit ec: ExecutionContext, appConfig: AppConfig)
-    extends FrontendController(mcc) with I18nSupport {
+    extends {
+  val cacheService = exportsCacheService
```

Comment:
Why you're assisgning this to cache service here? below you're using `exportsCacheService`

Reply:
Discussed, this val is necessary for trait